collector/packages.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'pganalyze Collector packages'
+backlink_href: /docs/collector
+backlink_title: 'pganalyze Collector'
+---
+
+import CollectorPkgInstallInstructions from '../components/CollectorPkgInstallInstructions'
+
+The collector is available on common Debian-based and Red Hat-based Linux distributions.
```

Comment:
Maybe "Pre-built collector packages are available on common Debian-based and Red Hat-based Linux distributions for x86-based architectures. Support for BSD-based platforms, or other architectures like ARM is available by [building from source](https://github.com/pganalyze/collector)."